content/events/events-snippet.md

```diff
@@ -1,3 +1,20 @@
 <!--This snippet goes above the main content in `events-content.md` in this folder-->
 
-EDGI's EEW events engage members of the public in the process of report-making, from delving into the data science to sharing stories about their home areas.
\ No newline at end of file
+EDGI's EEW events engage members of the public in the process of report-making, from delving into the data science and running jupyter notebooks to sharing stories about their home areas. Over the summer, we co-hosted workshops with the [Sunrise Movement Boston Hub](https://www.sunrisemovement.org/hubs/), as well as a four-part public workshop series dedicated to the completion of EEW Congressional Report Cards [link to reports page](https://edgi-govdata-archiving.github.io/EEW-Website/reports "Link to Reports Page") for two key committees. In commitment to the principles of EDJ, all of our code and data is available on [EDGI’s GitHub site](https://github.com/edgi-govdata-archiving "Link to EDGI's Github"), an open-source coding platform.
```

Comment:
Link formatting in Markdown is `[this is the text that will show up in blue](site/it/links/to)`